app/assets/stylesheets/preact/sidebar-widget.scss

```
@@ -23,7 +23,7 @@
               display: block;
               padding: 5px 0px;
               font-size: 18px;
-              color: $black;
+              color: var(--theme-color, $black);
```

```suggestion
              color: $black;
              color: var(--theme-color, $black);
```

Yup, as per @Glennmen's comment, it should look like this.

(Though we don't officially support any browser that _doesn't_ support CSS vars. So we might not need to worry about this in the future. That's a separate discussion.